### DISCUSSION TITLE
deepin terminal transparent background

### DISCUSSION THREAD

**Discussion by dacevedo12** [linuxdeepin/developer-center#3320] [Q&A | 问答板块] [answered]
The old deepin terminal supported transparency and blurred background. Are there any plans to restore it?

**Comment by BLumia** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10095765/180676807-df0ea3d1-b198-43b6-9f2f-f9c64f596ada.png)

The new terminal also supports transparency and blurred background. Please check the settings dialog and you need to apply these settings there.

**Reply by ghost** [reply]
> ![image](https://user-images.githubusercontent.com/10095765/180676807-df0ea3d1-b198-43b6-9f2f-f9c64f596ada.png)
> 
> The new terminal also supports transparency and blurred background. Please check the settings dialog and you need to apply these settings there.

Top bar doesn’t follow transparency and looks a bit out of context 👀 (suggestion for the future, top bar theme integration)
